literals/literals.go

```diff
@@ -116,41 +250,75 @@ func obfuscateLiterals(files []*ast.File, info *types.Info, blacklist map[types.
 		return true
 	}
 
-	key := genAesKey()
-	addedToPkg := false // we only want to inject the code and imports once
 	post := func(cursor *astutil.Cursor) bool {
 		switch x := cursor.Node().(type) {
-		case *ast.File:
-			if addedToPkg {
-				break
+		case *ast.Ident:
+			if x.Name == "true" || x.Name == "false" {
```

Comment:
this is buggy; imagine if I do:

```
true := 3
use(true)
```

You can't just look at the identifier's name. you have to use the type system to see if it is the builtin true/false value or not.